Quests/Buster Quests/quest_00300210.csv

```diff
@@ -34,20 +34,20 @@ OnDefencePhaseEnemyDecrease0#0,"""The enemy forces are weakening!<br>Let's take
 OnDefencePhaseEnemyDecrease1#0,"""There's only a few of them left!<br>Just a little more!"""
 OnWaveHapping#0,"""There's disturbing readings from the<br>Devil Castle! It seems it's going to<br>attack! Please watch yourselves!"""
 OnRamCountIncrease#0,"""We've collected more Mana.<br>You're able to summon more Busters!<br>Please continue to collect more Mana!"""
-OnBaseHpDecrease0#0,"""The $(0) Buster towers has been<br>damaged! Please help it!"""
-OnBaseHpDecrease1#0,"""The $(0) Buster towers doesn't look<br>very good! Please help save it!"""
-OnBrokenBase0#0,"""A Buster Tower has been destroyed!<br>We only have 3 remaining! We still<br>have leeway! Let's hurry and proceed!"""
+OnBaseHpDecrease0#0,"""The $(0) has been<br>damaged! Please help it!"""
```

Comment:
Don't remove "Buster towers" from this as this will show up as `The Orange has been damaged!`